deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -287,7 +288,6 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
-  replaces: smart-gateway-operator.v2.0.0
```

Comment:
Yea sorry it was in the last commit. As part of the bundle migration I'm going to set v2.0.1 as the "base" version. When I create a new SGO / SG release that `replaces` line will come back.

I was avoiding having to create bundles for all CSVs back to 0.2.0 since that kind of just seemed silly.